roottest/root/meta/rootcling/CMakeLists.txt

```diff
@@ -28,3 +28,7 @@ ROOTTEST_ADD_TEST(selectUnion
 ROOTTEST_ADD_TEST(ROOT10798
                   COMMAND ${ROOT_rootcling_CMD} -f ROOT10798Dict.cxx ${CMAKE_CURRENT_SOURCE_DIR}/ROOT10798LinkDef.h)
 
+# Issue #18833
+ROOTTEST_ADD_TEST(streamerInfoStdFunction
```

Comment:
Should we add a test that the dictionary can be loaded and has reasonable content?

Reply:
good point.